packages/lesswrong/components/posts/PostsAnnualReviewMarketTag.tsx

```diff
@@ -26,35 +68,34 @@ const PostsAnnualReviewMarketTag = ({ post, annualReviewMarketInfo, classes }: {
   classes: ClassesType<typeof styles>,
 }) => {
 
-  const { CommentsNode, LWPopper } = Components;
+  const { HoverOver, ContentStyles, ContentItemBody } = Components;
   const { anchorEl, hover, eventHandlers } = useHover();
 
-  const { annualReviewMarketComment } = post;
+  const year = annualReviewMarketInfo.year
+  const marketUrl = annualReviewMarketInfo.url
+
+  const golden = (highlightMarket(annualReviewMarketInfo)) ? "expectedWinner" : "expectedLoser"
```

Comment:
I'd probably call this `marketOutcomeClass`, or something more representative of what it's doing